src/mlpack/tests/main_tests/radical_test.cpp

```diff
@@ -120,7 +120,13 @@ BOOST_AUTO_TEST_CASE(RadicalBoundsTest)
  */
 BOOST_AUTO_TEST_CASE(RadicalDiffNoiseStdDevTest)
 {
-  arma::mat input = arma::randu<arma::mat>(5, 3);
+  std::string inputStr = "0.497369 0.891621 0.565789;";
```

Comment:
You could write:

```c++
arma::mat input("0.33821 0.494571 0.491079;"
                "0.33821 0.494571 0.491079;"
                "0.424898 0.297599 0.475061;");
```

to get the same result, this is fine, just looks somewhat unusual.

Reply:
Done.